src/t8_forest/t8_forest_cxx.cxx

```diff
@@ -1242,7 +1243,23 @@ t8_forest_element_point_batch_inside (t8_forest_t forest, t8_locidx_t ltreeid, c
   const t8_geometry_c *geometry = t8_cmesh_get_tree_geometry (cmesh, cgtreeid);
   T8_ASSERT (t8_geom_is_linear (geometry));
 #endif
+  if (geom_is_axis_aligned) {
```

Comment:
For now, we could use [this function](https://github.com/DLR-AMR/t8code/blob/main/src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.cxx#L82) to check if the geometry is axis aligned. For this, we can remove the debug macro around that function. This way, we get the geometry type directly from the tree and do not have to ask for it via a function argument.
And as soon as we merge PR #731, we can replace this with this [function](https://github.com/DLR-AMR/t8code/blob/feature-puma_patches_stretching/src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.hxx#L63-L67)

Reply:
Then I'll wait for #731 to merge!